gpkit/constraints/signomial_program.py

```diff
@@ -41,39 +35,21 @@ class SignomialProgram(object):
     >>> gp.solve()
     """
 
-    def __init__(self, cost, constraints, verbosity=2):
+    def __init__(self, cost, constraints, substitutions=None, verbosity=2):
```

Comment:
Naw, it's there noting that it's supplied as an argument to all program __init__s, and could be used in the future.